numpy/core/numeric.py

```diff
@@ -1902,18 +1902,18 @@ def fromfunction(function, shape, **kwargs):
     -----
     Keywords other than `dtype` are passed to `function`.
 
+    Function is only called once, not once per index.
```

Comment:
This feels redundant after your change above. Maybe swap the sequence here to:

```
the old misleading comment about (x,y,z), with a word like "usually" or "typically"
this comment

Notes
------
The new comment at the top
```

Reply:
I agree the new note is redundant. I really don't like the old ``fn(x, y, z)`` at coordinate ``(x, y, z)`` however as it is simply accurate. What do you think about removing the redundant comment in notes and replacing second line of the description with "The resulting array therefore has a value ``fn(*indices((x,y,z)))`` for an array of shape ``(x, y, z)``"?